mdagent/tools/base_tools/analysis_tools/descriptions.py

```diff
@@ -0,0 +1,43 @@
+TRAJECTORY_FILEID_DESC = "Trajectory File ID of the simulation to be analyzed"
```

Comment:
Sure, it was something i was trying to see if it would work, as some descriptions are common in a lot of tools: trajectory id, topology id, etc...
In the end, I'm not too convinced by it either, as I had to go and change them frequently during dev